packages/block-library/src/classic/edit.js

```diff
@@ -225,12 +226,17 @@ class ClassicEdit extends Component {
 const LazyClassicEdit = ( props ) => (
 	<LazyLoad
 		scripts={ [ 'wp-tinymce' ] }
-		onLoaded={ () =>
-			new Promise( ( resolve ) => {
-				window.wpMceTranslation();
-				resolve();
-			} )
-		}
+		onLoaded={ async () => {
+			const {
+				translations,
+				locale,
+				locale_script_handle: localeScriptHandle,
+			} = await apiFetch( { path: '/wp/v2/tinymce-i18n' } );
+
+			const { tinymce } = window;
+			tinymce.addI18n( locale, JSON.stringify( translations ) );
+			tinymce.ScriptLoader.markDone( localeScriptHandle );
+		} }
```

Comment:
@gziolo I think these changes will be interesting to you 😊 

cc @griffbrad because we'd also discussed using an endpoint to retrieve the translation json.